build/scripts/entrypoint-volume.sh

```diff
@@ -45,5 +45,19 @@ if [ -z "$CODE_HOST" ]; then
   CODE_HOST="127.0.0.1"
 fi
 
+# Update product.json with plug-in registry URL
+if [ -n "$CHE_PLUGIN_REGISTRY_URL" ]; then
+  # remove the suffix /v3 from this variable
+  CHE_PLUGIN_REGISTRY_ROOT_URL=${CHE_PLUGIN_REGISTRY_URL%/v3}
+
+  # Use OpenVSX of the plug-in registry
+  OPENVSX_URL="$CHE_PLUGIN_REGISTRY_ROOT_URL/openvsx/vscode"
+  sed -i -r -e "s|\"serviceUrl\": \"..*\"|\"serviceUrl\": \"${OPENVSX_URL}/gallery\"|" product.json
+  sed -i -r -e "s|\"itemUrl\": \"..*\"|\"itemUrl\": \"${OPENVSX_URL}/item\"|" product.json
+  sed -i -e "s|serviceUrl:\".*\",itemUrl:\".*\"},version|serviceUrl:\"${OPENVSX_URL}/gallery\",itemUrl:\"${OPENVSX_URL}/item\"},version|" vs/workbench/workbench.web.main.js
```

Comment:
When the initcontainer is stated I get in the logs:

```
sed: can't read vs/workbench/workbench.web.main.js: No such file or directory
```

and

```
$ docker run -ti --entrypoint cat quay.io/che-incubator-pull-requests/che-code:pr-85-amd64 /checode/checode-linux-libc/vs/workbench/workbench.web.main.js
cat: /checode/checode-linux-libc/vs/workbench/workbench.web.main.js: No such file or directory
```

Reply:
there is a missing `out` prefix 
`out/vs/workbench/workbench.web.main.js`